tests/registration/test_auth.py

```diff
@@ -0,0 +1,56 @@
+from allure import description, feature, step, title
+from hamcrest import assert_that, is_, none
+
+from framework.endpoints.authenticate_api import AuthenticateAPI
+from framework.steps.registration_steps import RegistrationSteps
+from framework.tools.generators import generate_string
+
+
+@feature("Authentication of user")
+class TestAuth:
+
+    @title("Failed authentication is limited in number")
+    @description(
+        "WHEN a user enters authentication data incorrectly more than 5 times, "
+        "THEN the service blocks the user for authentication"
+    )
+    def test_limitation_attempts(self, postgres, limit: int = 5):
+        with step("SetUp. Generation data of test user for registration"):
+            email = generate_string(length=2, additional_characters=["@te.st"])
+            first_name = generate_string(length=2)
+            last_name = generate_string(length=2)
+            password = generate_string(length=8, additional_characters=["@1"]).capitalize()
+
+            body = RegistrationSteps().data_for_sent(
+                email=email,
+                first_name=first_name,
+                last_name=last_name,
+                password=password,
+            )
+
+        with step("SetUp. Registration of test user"):
+            registrate = AuthenticateAPI().registration(body=body)
+            assert_that(registrate.status_code, is_(201), reason="Registration failed")
+
+        with step("User authentication attempt allowed number of times"):
+            for attempt in range(limit):
+                with step(f"Attempt {attempt}"):
+                    response = AuthenticateAPI().authentication(email=email, password=f"{password}1")
+                    assert_that(response.status_code, is_(401), reason="Invalid status code")
+                    assert_that(response.json()['data'], none(), reason="The 'data' field contains data")
+                    assert_that(
+                        response.json()['message'][0],
+                        is_(f"Invalid credentials for user's account with email = '{email}'"),
+                        reason="Authentication passed. There must be a mistake",
+                    )
+
+        with step("Authentication attempt over the limit"):
+            response = AuthenticateAPI().authentication(email=email, password=f"{password}1")
+            assert_that(response.status_code, is_(401), reason="Invalid status code")
+            assert_that(response.json()['data'], none(), reason="The 'data' field contains data")
+            assert_that(
+                response.json()['message'][0],
+                is_(f"User account with the email='{email}' have been locked "
```

Comment:
Failed with 
```
E           AssertionError: Authentication passed. There must be a mistake
E           Expected: "User account with the email='ae@te.st' have been locked out due to multiple failed login attempts"
E                but: was 'U'
```
May be we need to mark as failed or skip?

Reply:
We close it. We will discuss this in the conversation above